README.md

```diff
@@ -1 +1,184 @@
-# Hello
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [your_project_name] <a name="about-project"></a>
+
+**[Hello]** is a project to test Git and GitHub features and how to set up linters for HTML and CSS.
+
+## 🛠 Built With <a name="built-with"></a>
+HTML and CSS
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  HTML and CSS
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Basic project with HTML and CSS]**
+- **[Use of Linters]**
+- **[Use of Git and GitHub]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Great work on the previous reviewer's request! As he mention,  if you have decided not to deploy the application now, remove this section, or you can deploy it and add the deployed link in this section. If you're unsure about how to deploy the application, don't worry, you will learn it in a future project. In the meantime, feel free to delete this section and ensure that the 'live demo' is also removed from the **Table of Contents**.

Please remove the live demo section if the application won't be deployed.  
![image](https://github.com/wacoo/Hello-Microverse/assets/55840999/84580445-f6ed-431a-9f17-775311b7d719)  
![image](https://github.com/AdrianNv/Hello/assets/55840999/5cc81563-83ed-4435-8fc9-64c679c33813)